_content/articles/soteriological-purpose-of-nagarjunas-philosophy_william-l-ames.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "The Soteriological Purpose of Nagarjuna's Philosophy: A Study of Chapter Twenty-three of the Mūla-madhyamaka-kārikās"
+authors:
+  - "William  L.  Ames"
+external_url: "https://journals.ub.uni-heidelberg.de/index.php/jiabs/article/view/8738/2645"
+drive_links:
+  - "https://drive.google.com/file/d/10wTSD8BvGCZxubU7CNghpencgWjJfkNa/view?usp=sharing"
+course: epistemology
+tags:
+  - emptiness
+  - nibbana
```

Comment:
I think maybe vipassana instead? (more about method than result?) also, perhaps the `dialogue` tag